lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -68,9 +68,9 @@ def ship_to_hash
           'Address1'   => shipment.ship_address.address1,
           'Address2'   => shipment.ship_address.address2,
           'City'       => shipment.ship_address.city,
-          'State'      => shipment.ship_address.state.name,
+          'State'      => shipment.ship_address.state_name,
```

Comment:
Where is the corresponding change to the `ship_address` object? The extension shouldn't expect the host app to implement a `ship_address.state_name`.